prefetch.bs

```diff
@@ -71,38 +72,80 @@ Each {{Document}} has a <dfn export>prefetch buffer</dfn>, which is a [=list=] o
 A <dfn>prefetch record</dfn> is a [=struct=] with the following [=struct/items=]:
 * <dfn export for="prefetch record">URL</dfn>, a [=URL=]
 * <dfn export for="prefetch record">referrer policy</dfn>, a [=referrer policy=]
-* <dfn export for="prefetch record">response</dfn>, a [=response=]
+* <dfn export for="prefetch record">sandbox flags present</dfn>, a boolean
+* <dfn export for="prefetch record">redirect chain</dfn>, a [=list=] of [=responses=]
 * <dfn export for="prefetch record">expiry time</dfn>, a {{DOMHighResTimeStamp}}
 
+A [=prefetch record=]'s <dfn export for="prefetch record">response</dfn> is the last [=response=] in its [=prefetch record/redirect chain=], or null if that list [=list/is empty=].
+
 <p class="issue">Does the prefetch record need to contain redirect info, e.g. to verify that all redirects would pass `navigate-to`?</p>
```

Comment:
This probably can be updated now, since it does contain redirect info?

Reply:
Done.